warehouse/accounts/views.py

```diff
@@ -629,6 +630,8 @@ def register(request, _form_class=RegistrationForm):
         )
 
         send_email_verification_email(request, (user, email))
+        for email in user.emails:
+            send_new_email_added_email(request, (user, email))
```

Comment:
It might not make sense to send this during account registration -- the user is already receiving a verification email for their first and only email, so sending them another round of emails doesn't add any more information (and might confuse them about which email contains the verification link).

Reply:
Removing

Reply:
Agreed.